project-base/app/tests/FrontendApiBundle/Functional/Hreflang/HreflangLinksTest.php

```diff
@@ -63,6 +63,13 @@ public function getHreflangEntitiesDataProvider(): iterable
             'graphQlFileName' => 'BlogCategoryHreflangLinksQuery.graphql',
             'entityName' => 'blogCategory',
         ];
+
+        yield 'SeoPage' => [
+            'entityReference' => BlogArticleDataFixture::FIRST_DEMO_BLOG_SUBCATEGORY,
+            'routeName' => 'front_blogcategory_detail',
+            'graphQlFileName' => 'BlogCategoryHreflangLinksQuery.graphql',
+            'entityName' => 'blogCategory',
```

Comment:
is this correct? There should be `front_page_seo` route name, separate `graphQlFileName`, and `seoPage` under `entityName` key, right?

Reply:
No, no it's not correct... Thanks for noticing – 7d7daa237d